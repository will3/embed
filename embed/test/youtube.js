const test = require('./utils/test');

test('youtube',
  'https://www.youtube.com/watch?v=BBauxerc6TI');
