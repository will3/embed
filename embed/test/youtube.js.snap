// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` youtube 1`] = `
Object {
  "description": "South Park - Elon Musk Tesla D (South Park HandiCar) Elon Musk speaks to the media moments after breaking ground on a new commercial launch site in Brownsvil...",
  "originalUrl": "https://www.youtube.com/watch?v=BBauxerc6TI",
  "site_name": "YouTube",
  "thumbnail": Object {
    "url": "https://i.ytimg.com/vi/BBauxerc6TI/hqdefault.jpg",
  },
  "title": "Elon Musk FUNNIEST moments - YouTube",
  "video": Object {
    "description": undefined,
    "height": "360",
    "title": undefined,
    "url": "https://www.youtube.com/embed/BBauxerc6TI",
    "width": "640",
  },
}
`;
